src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -88,7 +88,7 @@ internal static class ParameterHelpers
                     Debug.Assert(addRefReadOnlyModifier, "If addReadonlyRef isn't true, we must have found a different location to encode the readonlyness of a function pointer");
                     ImmutableArray<CustomModifier> customModifiers = refKind switch
                     {
-                        RefKind.In => CreateInModifiers(binder, diagnostics, syntax),
+                        RefKind.In or RefKind.RefReadOnlyParameter => CreateInModifiers(binder, diagnostics, syntax),
```

Comment:
>RefKind.In or RefKind.RefReadOnlyParameter [](http://example.com/codeflow?start=24&length=42)

Since there is an open design question about metadata encoding for function pointers, consider adding a PROTOTYPE comment here.